src/test/ui/associated-const/issue-69020-assoc-const-arith-overflow.noopt.stderr

```diff
@@ -2,41 +2,41 @@ error: this arithmetic operation will overflow
   --> $DIR/issue-69020-assoc-const-arith-overflow.rs:29:22
    |
 LL |     const NEG: i32 = -i32::MIN + T::NEG;
-   |                      ^^^^^^^^^ attempt to negate with overflow
+   |                      ^^^^^^^^^ attempt to negate -2147483648 which would overflow
    |
    = note: `#[deny(arithmetic_overflow)]` on by default
 
 error: this arithmetic operation will overflow
   --> $DIR/issue-69020-assoc-const-arith-overflow.rs:31:35
    |
 LL |     const NEG_REV: i32 = T::NEG + (-i32::MIN);
-   |                                   ^^^^^^^^^^^ attempt to negate with overflow
+   |                                   ^^^^^^^^^^^ attempt to negate -2147483648 which would overflow
 
 error: this arithmetic operation will overflow
   --> $DIR/issue-69020-assoc-const-arith-overflow.rs:34:22
    |
 LL |     const ADD: i32 = (i32::MAX+1) + T::ADD;
-   |                      ^^^^^^^^^^^^ attempt to add with overflow
+   |                      ^^^^^^^^^^^^ attempt to compute `2147483647 + 1` which would overflow
```

Comment:
So computations are enclosed in `` ` `` but constants are not? Are we doing that consistently?

Reply:
no